src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.ts

```diff
@@ -6,7 +6,7 @@ import { TILE_ACTION_TYPES, SpotTileActions } from '../actions'
 import { of, timer } from 'rxjs'
 import { RfqRequest } from '../model/rfqRequest'
 import { SpotTileState } from '../spotTileReducer'
-import { CurrencyPairState } from '../../../../src/shell/referenceData'
+import { CurrencyPairState } from '../../../data/referenceData'
```

Comment:
This import is strange. its going up 3 levels. Maybe we need to move CurrencyPair State